cli/service/service.go

```diff
@@ -76,6 +82,84 @@ func List() {
 	fmt.Printf("%s\n", jsonBytes)
 }
 
+func Log(ServiceName string) {
+	msgPrinter := i18n.GetMessagePrinter()
+	ref_url := "\"ref_url\": \"" + ServiceName + "\""
+	re := regexp.MustCompile(`instance_id\": \"(.*?)\"`)
+	var inputs api.AllServices
+	cliutils.HorizonGet("service", []int{200}, &inputs, false)
+	_, err := cliutils.DisplayAsJson(inputs.Instances["active"])
```

Comment:
Why are you processing the response as JSON? You should be using the returned object directly and then iterating through the structure to find the service you're looking for.